content/en/docs/reference/commands/jx_edit_buildpack.md

```diff
@@ -5,8 +5,25 @@ slug: jx_edit_buildpack
 url: /commands/jx_edit_buildpack/
 description: list of jx commands
 ---
+## Customising buildpacks for boot based clusters
+
+If you want to specify a custom buildpack for a boot based cluster. Then you
+have to add the following fields to the jx-requirements.yml file before running
+boot. All the fields have to be added together otherwise jx won't be able to
+figure out where to get the buildpack from.
+
+```yaml
+buildPacks:
+  buildPackLibrary:
+    name: "Test name"
+    gitURL: "github.com/jx-user/test-repo"
+    gitRef: "master"
+```
+
 ## jx edit buildpack
 
+**This command only works for clusters created with install rather then boot.**
```

Comment:
That actually should be added in the jx codebase. This should be shown to the user when he uses the CLI.